dbc2val/Readme.md

```diff
@@ -104,18 +104,26 @@ A smaller excerpt from the above sample, with less signals.
-| VEHICLEDATABROKER_DAPR_APP_ID | -               | -                    | VEHICLEDATABROKER_DAPR_APP_ID | -                     | Add dapr-app-id metadata                                                                                |
+| parameter                     | default value    | config file                   | Env var      | command line argument | description                                                                                             |
+|-------------------------------|------------------|-------------------------------|--------------|-----------------------|---------------------------------------------------------------------------------------------------------|
+| config                        | -                | -                             | -            | `--config`           | Configuration file  |
+| dbcfile                       | -                | [can].dbc                     | DBC_FILE     | `--dbcfile`          | DBC file used for parsing CAN traffic |
+| dumpfile                      | -                | [can].candumpfile             | CANDUMP_FILE | `--dumpfile`         | Replay recorded CAN traffic from dumpfile |
+| canport                       | -                | [can].port                    | CAN_PORT     | `--canport`          | Read from this CAN interface  |
+| use-j1939                     | False            | [can].j1939                   | USE_J1939    | `--use-j1939`        | Use J1939. See below for Env var usage |
+| use-socketcan                 | False            | -                             | -            | `--use-socketcan`    | Use SocketCAN (overriding any use of --dumpfile) |
+| mapping                       | mapping/vss_3.1.1/vss_dbc.json | [general].mapping | MAPPING_FILE | `--mapping`        | Mapping file used to map CAN signals to databroker datapoints. Take a look on usage of the mapping file |
+| strict                        | False            | -                             | -            | `--lax-dbc-strict`  | Disable strict parsing of dbc file |
+| server-type                   | kuksa_val_server | [general].server_type         | SERVER_TYPE  | `--server-type`     | Which type of server the feeder should connect to (kuksa_val_server or kuksa_databroker |
+| ip (kuksa_databroker)         | 127.0.0.1        | [kuksa_databroker].ip         | VDB_ADDRESS,<br /> DAPR_GRPC_PORT | -| Databroker IP address, See below for Env var usage |
+| port (kuksa_databroker)       | 55555            | [kuksa_databroker].port       | VDB_ADDRESS,<br /> DAPR_GRPC_PORT | -| Databroker port, See below for Env var usage |
+| protocol (kuksa_databroker)   | grpc             | [kuksa_databroker].protocol   | -            | -                     | Configuration currently ignored |
+| insecure (kuksa_databroker)   | True             | [kuksa_databroker].insecure   | -            | -                     | Configuration currently ignored |
```

Comment:
No when we are to add authorization for data broker I think that "insecure" is a bad name to control TLS authentication.

Would it possible be better to rename it to `tls_auth` or similar? And if we want certificates/keys to be explicitly named - what is it that we want to provide? Just a folder-name and expected keys/certs to have specific names, or individual. If we currently just (want to) support client auth of server (i.e. not mutual auth) - then do we need anything more than a single file for the root cert? Which potentially could be used to imply if "insecure" should be used

And for Token authorization - there the client need only a single token file. 

So an idea I have - remove "insecure" and instead add two configs (with empty default)

- `tls_root_ca` - Path to root CA, if not given or empty "insecure" mode will be used
- `client_token` - If not given or empty the client will not try to authorize against broker/server

Reply:
`client-token`(with dash for consistency), or actually maybe just `token`(there is noch such thing as a server token in the system anyway) seems useful

For TLS, instead of "`--insecure` maybe just `--no-tls`

Insecure mode should not be used when when no root cert is given, because the dafualt behavior should be, that hte TLS implementation refers to the system Root certs (as it will do "out of the box" anyway) 

So, even without `--tls-ca-cert` it should attempt tls connection unless `--no-tls`is given


Also, I think the plumbing for optionally giving custom root-ca-certs  needs to be done in kuksa-client. An alternative could be not implementing that, and instead say, people should configure the root-ca store of their system accordingly, but on the other that makes testings very unpractical.

Reply:
@SebastianSchildt -digging a bit deeper into code it appears that these configs actually are passed "as is" to kuksa-client in https://github.com/eclipse/kuksa.val/blob/master/kuksa-client/kuksa_client/cli_backend/__init__.py#L25, so we cannot easily change name in the feeders. It also the same file that specifies that there shall be a config "protocol"